download.html

```diff
@@ -20,6 +20,17 @@
 <div class="container">
     <div class="download_main">
 
+    <h2>Installing with go-xcat <sup class="new">New!</sup> </h2>
+
+    To easily install xCAT and handle the dependency package, a new script <a href=http://xcat.org/files/go-xcat>go-xcat</a> is provided.
```

Comment:
And I think we can assume that the end user of xCAT has the basic understanding of the command ``wget``, and how ``wget`` works. Anyway, the URL is there, included in the command line already, the end user can treat it with anyway he/she like.